quickstarts/maven/docker-file-provided/pom.xml

```diff
@@ -103,7 +103,8 @@
                                 <image>
                                     <name>jkube/docker-file-provided</name>
                                     <build>
-                                        <dockerFile>subdir/Dockerfile</dockerFile>
```

Comment:
I **think** this is related to what we discussed during the stand-up, this example highlights how the contextDir is computed from the actual Dockerfile location.

Example expects no `contextDir`to be defined and also Dockerfile to be located under `src/main/docker`
https://github.com/eclipse/jkube/tree/a32319023d658ba8a646bf63dffc08b18944ffe9/quickstarts/maven/docker-file-provided#contextdir-and-dockerfile